### PR TITLE
partner_routing to trip completion workflow

### PR DESCRIPTION
**Pull request by premkumar91**

### CODE REVIEW

File: src/component/PartnerRoutingToDeliveryPoint.js [hunk 1]
```diff
@@ -11,6 +12,13 @@ const PartnerRoutingToDeliveryPoint = () => {
         tripId: trip_id
     } = useParams();
 
+    const completeTrip = async () => {
+      const redirectUrl = `http://localhost:4000/partner/${partner_id}/initiateTripCompletion/${trip_id}`;
+      window.setTimeout(() => {
+        window.location.href = redirectUrl;
```
Comment (SanthoshGugan) [inline src/component/PartnerRoutingToDeliveryPoint.js:18]: https://reactrouter.com/en/main/hooks/use-navigate
we can have useNavigate instead direct window navigate, app will not refresh

File: src/component/PartnerReachedRequestPoint.js [hunk 2]
```diff
@@ -20,10 +20,13 @@ const PartnerReachedRequestPoint = () => {
         const fetch = async () => {
             const result = await partnerReachedRequestPointApi(req);
             setIsComplete(true);
+            window.setTimeout(()=>{
+                const redirectUrl = `http://localhost:4000/partner/${partner_id}/getRoutingToDeliveryPoint/${trip_id}`;
+                window.location.href = redirectUrl;
```
Comment (SanthoshGugan) [inline src/component/PartnerReachedRequestPoint.js:25]: https://reactrouter.com/en/main/hooks/use-navigate
we can have useNavigate instead direct window navigate, app will not refresh

File: src/component/PartnerRoutingToRequestPoint.js [hunk 3]
```diff
@@ -11,6 +13,11 @@ const PartnerRoutingToRequestPoint = () => {
         tripId: trip_id
     } = useParams();
 
+    const partnerReached = async() => {
+      const redirectUrl = `http://localhost:4000/partner/${partner_id}/reachedRequestPoint/${trip_id}`;
+      window.location.href = redirectUrl;
```
Comment (SanthoshGugan) [inline src/component/PartnerRoutingToRequestPoint.js:18]: https://reactrouter.com/en/main/hooks/use-navigate
we can have useNavigate instead direct window navigate, app will not refresh

File: src/component/PartnerRoutingToDeliveryPoint.js (hunk 1 above)
Comment (premkumar91) [reply src/component/PartnerRoutingToDeliveryPoint.js]: done

File: src/component/PartnerReachedRequestPoint.js (hunk 2 above)
Comment (premkumar91) [reply src/component/PartnerReachedRequestPoint.js]: done

File: src/component/PartnerRoutingToRequestPoint.js (hunk 3 above)
Comment (premkumar91) [reply src/component/PartnerRoutingToRequestPoint.js]: done

File: src/component/PartnerRoutingToRequestPoint.js [hunk 4]
```diff
@@ -1,15 +1,24 @@
 import React, {useState,useEffect, useRef} from "react";
 import { Loader } from "@googlemaps/js-api-loader"
 import { fetchTripDetailsAndRouting } from "../utils/locationUtil";
-import { useParams } from "react-router-dom";
-import { apiKey } from "../config/config";
```
Comment (SanthoshGugan) [inline src/component/PartnerRoutingToRequestPoint.js:5]: Buddy, config moved to config file, no need to replace to dummy value each time. please revert apiKey import for all the places

**Review by SanthoshGugan — APPROVED** [review]
LGTM!

**Pull request merged**